docs/SUMMARY.md

```diff
@@ -8,4 +8,5 @@
   - [IRC Modes](./irc_modes.md)
 - [Setup](./bridge_setup.md)
 - [Administrators Guide](./administrators_guide.md)
-- [IRC Operators](./irc_operators.md)
\ No newline at end of file
+- [IRC Operators](./irc_operators.md)
+- [Bridged Networks](./bridged_networks.md)
```

Comment:
I'd hoist this entry right below Usage so users see it.

Reply:
dome